src/unittests/test_group.spec.ts

```diff
@@ -334,10 +335,10 @@ g.test('shouldReject').fn(async t0 => {
 
   const g = makeTestGroupForUnitTesting(UnitTest);
 
-  g.test('a').fn(async t => {
+  g.test('a').fn(t => {
     t.shouldReject(
       'Error',
-      (async () => {
+      (() => {
```

Comment:
This one is changing the meaning of the test. Before it tests shouldReject makes the test 'a' fail, now I'm pretty sure shouldReject is not getting called at all and it's testing that if the test fn throws then 'a' fails

Reply:
Ah yea good catch, I think I forgot to revert this one after putting the disable for the lint in this file. Thanks!